src/core.jl

```diff
@@ -90,16 +110,19 @@ end
     ellipse = Ellipse(center, ρx, ρy)
 
 A `Drawable` ellipse with center `center` and parameters `ρx` and `ρy`
+with keyword arguments `thickness` and `fill` for drawing hollow ellipse
 
 """
 struct Ellipse{T<:Real, U<:Real} <: Drawable
     center::Point
     ρx::T
     ρy::U
     thickness::Int
-	fill::Bool
+    fill::Bool
 end
 
+Ellipse(center::Point, ρx::T, ρy::U; thickness::Int =  0, fill::Bool = true) where {T<:Real, U<:Real} = Ellipse(center, ρx, ρy, thickness, fill)
```

Comment:
It's unlikely to reformat the code after being merged so we need to make sure that we do it correctly here. There're still some other similar format issues.

```suggestion
Ellipse(center::Point, ρx::T, ρy::U; thickness::Int = 0, fill::Bool = true) where {T<:Real, U<:Real} = Ellipse(center, ρx, ρy, thickness, fill)
```